newIDE/app/src/SceneEditor/index.js

```diff
@@ -366,6 +366,32 @@ export default class SceneEditor extends Component {
     );
   };
 
+  _onObjectSelected = selectedObjectName => {
+    this.setState({
+      selectedObjectNames: [selectedObjectName],
+    });
+  };
+
+  _onAddInstanceUnderCursor = () => {
+    const objectSelected = this.state.selectedObjectNames[0];
+    const cursorPosition = this.editor.getLastCursorPosition();
+    this._addInstance(cursorPosition[0], cursorPosition[1], objectSelected);
+    this.setState({
+      selectedObjectNames: [objectSelected],
+    });
+  };
+
+  shortenedString = (inputString: string, maxlength: number) => {
```

Comment:
This is a function doing some action, so I would rather name it "shortenString" or "makeEllipsis". 
And put it in an external file in `Utils/` folder and import it here :)

Reply:
Good idea, the functionality might come in handy in other areas of the editor :)

Reply:
How should we call the file? :) Utils/UI.js?

Reply:
UI.js is too broad, either like the function name `MakeEllipsis.js`, `ShortenString.js` or something a bit more general like `StringHelpers.js`.

Reply:
done :)